avro-schema/sum-part.avsc

```diff
@@ -43,12 +48,12 @@
             "doc": "If specified, the hamming weight of the vector will be verified during the validity check on the server."
         },
         {
-            "name": "sums",
+            "name": "sum",
```

Comment:
The cardinality of uuids and sum have to be same, is it not ?

Reply:
Should this an array of (sub) records [(uuid, sum)] ?

Reply:
We actually want to compute only one "sum" over many `batch_uuid`'s, and this is important for the privacy story of Prio. In other words, we want to sum the `sum`'s of your subrecords and only reveal the total.

Reply:
Then should this be an array? The change from sums to sum seemed to indicate what you mentioned but the schema still refers to an array?

Reply:
@yuriks and I also found this confusing when we hashed it out on Friday afternoon. Here's how I think about it: the basic atom of data produced by any client in this system is a vector of _bins_ fields (_bins_ is also _dim_ or _dimension_ in some contexts). When we sum up data, the result is still a _bins_-dimensional vector. I changed this to `sum`, singular to indicate it's a single vector, but by dint of being a vector, it's still an array of longs.

Reply:
Thanks for the clarification and the addition in the doc field for the same.